website/src/graphql/wordpress/queries.graphql

```diff
@@ -34,3 +34,23 @@ query MainMenu {
     }
   }
 }
+
+query MenuByID($slug: Int!) {
+  menus(where: { id: $slug }) {
+    nodes {
+      databaseId
+      menuItems(where: { parentDatabaseId: 0 }) {
```

Comment:
Hmm why are you using `parentDatabaseId: 0` here? Also, I would rather us have one query definition for both menus instead of a separate one for the main site menu.

Reply:
I don't know why but including it makes the graphql query return Contact-Us, but excluding it means Contact-Us does not appear. I can't tell why, as the parentDatabaseID and parentID themselves are not different from other nodes. So, including this (where: { parentDatabaseId: 0 }) was the simplest way to bring back Contact-Us.